### DISCUSSION TITLE
能否支持deeplx(deepl的逆向接口)

### DISCUSSION THREAD

**Discussion by Micah123321** [immersive-translate/immersive-translate#535] [Ideas]
### 新功能 | New Feature

1.  [docker镜像](https://hub.docker.com/r/zu1k/deepl) 支持这个docker 镜像部署
2. 为deepl的逆向接口 响应速度很快
3. 希望在设定的时候只填写接口url

### 为什么您想要添加这些功能 | Why you want to add these features

1. deepl的api注册需要国外银行卡,并且国内属地的不行
2. 自行提供docker镜像以及服务器

### 提议 | Proposal

请求格式为  Request
Method: POST
Path: /translate
Content-Type: application/json
{
    "text": "text to be translated",
    "source_lang": "auto",
    "target_lang": "ZH"
}     具体可以看docker镜像的文档

**Comment by theowenyoung** [comment]
开发者设置 开 Beta 即可看到

**Reply by Micah123321** [reply]
> 开发者设置 开 Beta 即可看到

谢谢!
